tests/scripts/remote_monitoring_tests/scalability_test.md

```diff
@@ -48,6 +49,23 @@ cd <KRUIZE_REPO>/tests/scripts/remote_monitoring_tests/scale_test
 
 ```
 
+To test namespace experiments:
+
+```
+cd <KRUIZE_REPO>/tests/scripts/remote_monitoring_tests/scale_test
+./remote_monitoring_tests/scale_test/remote_monitoring_scale_test_bulk.sh -i quay.io/kruize/autotune_operator:0.0.20_mvp  -u 250  -d 15  -n 20 -t 6  -q 10  -s 2023-01-10T00:00:00.000Z  -r /tmp/scale_test_results -c namespace
+
+```
+
+To test with a mix of container & namespace experiments:
+
+```
+cd <KRUIZE_REPO>/tests/scripts/remote_monitoring_tests/scale_test
```

Comment:
I guess the command here in all 3 places needs to updated to start from the `./remote_monitoring_scale_test_bulk.sh -i ... ` since we have already mentioned above to switch to `scale_test` directory.